resources/kcp/charts/mothership-reconciler/templates/service.yaml

```diff
@@ -16,5 +16,9 @@ spec:
       targetPort: {{ .Values.service.istioPort }}
       protocol: TCP
       name: istio-http
+    - port: {{ .Values.fluentbit.port }}
+      targetPort: fluentbit
```

Comment:
I would prefer to call http-metrics so that it would make the purpose of the port clear

Reply:
I think http-metrics is misleading since reconciler also opens up metrics

Reply:
eg through http, we also expose the endpoint `/metrics`